Jenkinsfile_nightly

```diff
@@ -35,20 +33,25 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
     ]
 }
 
-// Vars for Kubernetes
-env.IDAM_API_BASE_URI = 'https://preprod-idamapi.reform.hmcts.net:3511'
-env.S2S_BASE_URI = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
-env.IDAM_WEBSHOW_WHITELIST = 'https://em-show-aat.service.core-compute-aat.internal/oauth2/callback'
-env.DM_STORE_APP_URL = 'http://dm-store-aat.service.core-compute-aat.internal'
-
-
 def vaultOverrides = [
         'preview' : 'aat',
         'spreview': 'saat'
 ]
 
 withNightlyPipeline(type, product, component) {
+    // Vars for Kubernetes
+    env.IDAM_API_BASE_URI = 'https://idam-api.aat.platform.hmcts.net'
+    env.OPEN_ID_API_BASE_URI = 'https://idam-web-public.aat.platform.hmcts.net/o'
+    env.OIDC_ISSUER_BASE_URI = 'https://forgerock-am.service.core-compute-idam-aat.internal:8443/openam/oauth2/hmcts'
+    env.S2S_BASE_URI = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
+    env.IDAM_WEBSHOW_WHITELIST = 'https://em-show-aat.service.core-compute-aat.internal/oauth2/callback'
+    env.DM_STORE_APP_URL = 'http://dm-store-aat.service.core-compute-aat.internal'
+
+    env.TEST_URL = "http://em-anno-aat.service.core-compute-aat.internal"
+
     overrideVaultEnvironments(vaultOverrides)
     loadVaultSecrets(secrets)
-    installCharts()
+
+    enableMutationTest()
```

Comment:
Have you provided the pitest task in gradle to make this mutation work?  If not enablemutationtest() will fail.

Reply:
Have removed this for time been.